### DISCUSSION TITLE
Description as field

### DISCUSSION THREAD

**Discussion by SaurabhKumarJindal** [ironSource/license-report#180] [Ideas]
shall be added package description as field to have complete license information about the packages.

I think, it would be great to have an field option 'description'. if there is not available for the package then can empty or n\a.

**Comment by BePo65** [comment]
If I'm right, any field in the package.json file can be used as output field by using the configuration (see section [Select fields for output](https://github.com/kessler/license-report?tab=readme-ov-file#select-fields-for-output)).

In your case the following configuration should do the trick:
```
"description": {
    "value": 'n/a',
    "label": 'Description'
  },
"fields": [
    "name",
    "installedVersion",
    :
    :
    "description"
  ]
```

Please let me know, if I misunderstood your requirement or if my proposition does not work.

**Reply by SaurabhKumarJindal** [reply]
how to extract the description from package?. we also need an extractDescription.js file in the 'lib' folder as working tools. what do you think?

**Comment by SaurabhKumarJindal** [comment]
your suggestion is working. thanks

**Comment by SaurabhKumarJindal** [comment]
your suggestion is working. thanks
